src/01-general/history.apib

```diff
@@ -0,0 +1,69 @@
+## History [/history]
+
+### history.list [GET /history.list]
+
+Retrieve a list of records about an item.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object)
+            + about (object, required)
+                + type (enum[string])
+                     + Members
+                         + contact
+                         + company
+                + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string)
+                + title: `Marc was on a call with John from this company` (string)
+                + description: `An extended description` (string)
+                + recorded_at: `2017-05-09T11:25:11+00:00` (string)
+                + type (enum[string])
+                    + Members
+                        + call
+                        + client_form_filled_out
+                        + client_satisation_mail_sent_project
+                        + client_satisation_mail_sent_ticket
+                        + company
+                        + contact
+                        + creditnote
+                        + custom
+                        + deal
+                        + edit
+                        + email
+                        + event
+                        + external_interaction
+                        + file
+                        + gmail_email
+                        + invoice
+                        + linkedin
+                        + mail_sent
+                        + mailing
+                        + meeting
+                        + note
+                        + order
+                        + permanent_note
+                        + product
+                        + project
+                        + sale
+                        + ticket_closed
+                        + timetracking
+                        + todo
+                + relates_to (object)
+                    + (object)
+                        + type (enum[string])
+                            + Members
+                                + contact
+                                + company
+                                + deal
```

Comment:
What should we do when the related resource was deleted? Should we just set `relates_to` to `null`? Should `relates_to` have an extra field `deleted`? Another option?

Reply:
I think the `id` should be `null` to make clear that you can no longer reference the entity, eventually with an added `(is_)deleted` property.